Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/openSAML/extraction/OpenSAMLAssertionExtractorImpl.java

```diff
@@ -470,7 +469,7 @@ private static void populatePurposeOfUseAttribute(final Attribute attribute, fin
      */
     private static AssertionType initializeAssertion() {
 
-        LOG.debug("Initializing Assertion to Default: " + EMPTY_STRING);
+        LOG.debug("Initializing Assertion to Default: ");
```

Comment:
This makes sense if you're reading it in the code, but thing about how it will look in the log:

> Initializing Assertion to Default:

It might be more useful for debugging purposes to explicitly call out that the default is blank, e.g.,
* "Initializing Assertion to default (no value)"
* "Creating default (empty) assertion"